src/test/bundler_test.ts

```diff
@@ -44,12 +49,23 @@ suite('Bundler', () => {
         }
         bundler = new Bundler(bundlerOpts);
         const manifest = await bundler.generateManifest([inputPath]);
-        const documents = await bundler.bundle(manifest);
+        const bundleResult = await bundler.bundle(manifest);
+        documentBundle =
+            bundleResult.manifest.getBundleForFile(inputPath)!.bundle;
+        const {documents} = bundleResult;
         return documents.get(inputPath)!.ast;
       }
 
   suite('Default Options', () => {
 
+    test('URLs for inlined HTML imports are recorded in Bundle', async () => {
+      await bundle(inputPath);
+      assert.sameMembers(
+          [...documentBundle.inlinedHtmlImports],
+          ['imports/simple-import.html'],
+          [...documentBundle.inlinedHtmlImports].join(', '));
```

Comment:
Definitely no on the console.log. Looking around in the chai repo, someone [may have added diffing to sameMembers](https://github.com/chaijs/chai/pull/863)? Two more things to try, and if they don't work your solution is good:

1) Update chai to latest version
2) Set `chai.config.showDiff = true;`